tomviz/modules/Module.cxx

```diff
@@ -285,6 +294,10 @@ bool Module::deserialize(const QJsonObject& json)
     setUseDetachedColorMap(useDetachedColorMap);
   }
 
+  if (json.contains("activeScalars")) {
```

Comment:
I am a little torn on whether activeScalars is a little vague, consider activeScalarArray. Would be a pain to rename, I think we should think about the name before it gets too ingrained, but we can change it in a follow up commit.